src/ubuntu/18.04/cross/arm-alpine/hooks/pre-build

```diff
@@ -2,4 +2,4 @@
 
 SCRIPT=$(readlink -f "$0")
 SCRIPTPATH=$(dirname "$SCRIPT")
-$SCRIPTPATH/../../../../build-scripts/run-arcade-build-rootfs.sh ubuntu-18.04 alpine arm lldb3.9
+$SCRIPTPATH/../../../../build-scripts/run-arcade-build-rootfs.sh ubuntu-18.04 alpine3.14 arm lldb3.9
```

Comment:
How was `alpine` mapped to actual alpine version before this change?

Reply:
The default is 3.13: https://github.com/dotnet/arcade/blob/12b42d0d77948889890ef2f35af8c46438d3f2d2/eng/common/cross/build-rootfs.sh#L258-L262.

I think we should also omit `numactl-dev` from alpine 3.13 there (as we omit `libunwind8-dev` from riscv64), so if anyone else is using rootfs with 3.13 their installation don't break.

Reply:
ok